apps/explorer/components/SNIFilter.vue

```diff
@@ -1,216 +1,12 @@
-          placeholder="Search Name or Id..."
-          required
-        />
-      </div>
-      <div class="flex w-full justify-end gap-2">
-        <select class="select max-w-xs rounded-full bg-neutral text-white">
-          <option disabled selected class="text-white">
-            Conservation status
-          </option>
-          <option>Preserved specimen</option>
-          <option>Human observation</option>
-          <option>Machine observation</option>
-        </select>
-        <select class="select max-w-xs rounded-full bg-neutral text-white">
-          <option disabled selected class="text-white">Location</option>
-          <option>Kenya</option>
-          <option>Netherlands</option>
-        </select>
-        <div class="relative w-1/2 rounded-full bg-neutral">
-          <div
-            class="pointer-events-none absolute inset-y-0 left-0 flex items-center rounded-full bg-inherit p-3"
-          >
-            <svg
-              aria-hidden="true"
-              class="h-5 w-5 text-white"
-              fill="currentColor"
-              viewBox="0 0 20 20"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                fill-rule="evenodd"
-                d="M6 2a1 1 0 00-1 1v1H4a2 2 0 00-2 2v10a2 2 0 002 2h12a2 2 0 002-2V6a2 2 0 00-2-2h-1V3a1 1 0 10-2 0v1H7V3a1 1 0 00-1-1zm0 5a1 1 0 000 2h8a1 1 0 100-2H6z"
-                clip-rule="evenodd"
-              ></path>
-            </svg>
-          </div>
-          <input
-            datepicker
-            type="text"
-            class="block h-full w-full rounded-full bg-inherit pl-12 text-sm text-white placeholder:text-white focus:outline-none"
-            placeholder="Created at"
-          />
-        </div>
-        <div class="relative w-1/2 rounded-full bg-neutral">
-          <div
-            class="pointer-events-none absolute inset-y-0 left-0 flex items-center rounded-full bg-inherit p-3"
-          >
-            <svg
-              aria-hidden="true"
-              class="h-5 w-5 text-white"
-              fill="currentColor"
-              viewBox="0 0 20 20"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                fill-rule="evenodd"
-                d="M6 2a1 1 0 00-1 1v1H4a2 2 0 00-2 2v10a2 2 0 002 2h12a2 2 0 002-2V6a2 2 0 00-2-2h-1V3a1 1 0 10-2 0v1H7V3a1 1 0 00-1-1zm0 5a1 1 0 000 2h8a1 1 0 100-2H6z"
-                clip-rule="evenodd"
-              ></path>
-            </svg>
-          </div>
-          <input
-            datepicker
-            type="text"
-            class="block h-full w-full rounded-full bg-inherit pl-12 text-sm text-white placeholder:text-white focus:outline-none"
-            placeholder="Updated at"
-          />
-        </div>
-      </div>
-    </form>
+    <FilterMobile></FilterMobile>
+    <FilterDesktop></FilterDesktop>
```

Comment:
Inside the components itself.

**FilterDesktop**
_<form class="hidden w-full py-12 lg:flex">_

**FilterMobile**
_<div
    tabindex="1"
    class="collapse collapse-arrow bg-neutral text-white lg:hidden"
  >_
  
  
  If you wish I can change for the parent components but it will be six and half a dozen.

Reply:
After some reading the article mentioned in the previous post and doing some research I think it's an overthinking of the solution suggested as mentioned in the link:
**Old Trusty - User Agent Parsing**
_When I mentioned this was "ancient technology" I wasn't too far from the truth, user agents have been sent from browsers since the beginning of the modern internet._

Yet in the User Agent, the library mentioned in the website _mobile-detect.js_ do not recommend this method. as follows:
**Warning**
_TL;DR: you should not use this library in your HTML page and it's less reliable when used server-side (Node.js)
As mentioned later, "User-Agent" based detection is not a reliable solution in most cases, because:

- The rules (regular expressions) are constantly out-dated and incomplete
- You have to update the detection code continuously
- There are other ways to detect how your web application should behave:
- feature detection, e.g. [Modernizr](https://modernizr.com/)
- **[Media Queries](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Media_queries), examples at https://mediaqueri.es/**
- Maybe there are some libraries out there (which are probably not free) doing a more reliable job
_

The other solution suggested is Client Hints, which imo has red flags for implementing in our solution:

- Only supported by Chrome/Edge/Opera
- Values are rounded/estimated to prevent device fingerprinting


Media queries is already the method implement in our solution and working, using Tailwind CSS breaking points.

Reply:
Besides that, I think this responsivity issue is not related with the PR or the task, as it was the implementation of the Datepicker in the application.

Reply:
I think we can merge this one and then implement https://www.npmjs.com/package/@nuxtjs/device when it will be supported by Nuxt3.

Yes, we can open another PR for conditional rendering.